lgc/test/BuiltIns/cs-subgroupsize.lgc

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py UTC_ARGS: --tool lgc
-; RUN: lgc -mcpu=gfx802 - < %s | FileCheck --check-prefixes=WAVE64 %s
+; RUN: lgc -mcpu=gfx1010 - < %s | FileCheck --check-prefixes=WAVE64 %s
 ; RUN: lgc -mcpu=gfx1010 - < %s | FileCheck --check-prefixes=WAVE32 %s
```

Comment:
Same here: only one RUN line for gfx1010.

Reply:
Fixed.